src/main/java/seedu/ui/UserInputTextUi.java

```diff
@@ -0,0 +1,22 @@
+package seedu.ui;
+
+import java.util.Scanner;
+
+public class UserInputTextUi {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static String getUserInput() {
+        String userInput = scanner.nextLine().trim();
+        if (userInput.contains(",")) {
+            String newUserInput = userInput.replace(",", "");
+            ExceptionTextUi.forbiddenInputCommaMessage(newUserInput);
+            return newUserInput;
+        }
+        return userInput;
+    }
+
+    public static String getUserDeleteConfirmation() {
+        String userDeleteConfirmation = scanner.nextLine().trim();
+        return userDeleteConfirmation;
+    }
```

Comment:
@lezongmun I think this method can be used for any confirmation, should this be changed to just `getUserConfirmation`?